components/builder-originsrv/src/data_store.rs

```diff
@@ -50,6 +50,7 @@ impl DataStore {
         // The order here matters. Once you have deployed the software, you can never change it.
         migrations::next_id::migrate(&mut migrator)?;
         migrations::origins::migrate(&mut migrator)?;
+        migrations::origin_public_keys::migrate(&mut migrator)?;
```

Comment:
This is fine right now, because we've never deployed this code base. In the future, you can't change the order of these lines without breaking the migrations. :)